### ISSUE TITLE
Can't resolve 'camel-case'

### ISSUE THREAD

**Issue by elado** [wmira/react-icons-kit#9]
```
     @ ./~/react-icons-kit/SvgIcon.js 18:17-38
     @ ./~/react-icons-kit/Icon.js
     @ ./~/react-icons-kit/index.js
```

`camel-case` is defined as a devDependency.

**Comment by wmira** [comment]
Hi @elado ,

I published a fix to this @1.0.7 https://codesandbox.io/s/QWvNmvnzq

Thanks for the report.

**Comment by elado** [comment]
Version here on GitHub is still 1.0.6

`SvgIcon` uses `camel-case`.
https://github.com/wmira/react-icons-kit/blob/master/src/SvgIcon.js#L4

It's defined in `devDependencies`, should be moved to `dependencies`. Same for `prop-types`.
